src/coreclr/src/jit/lower.cpp

```diff
@@ -3688,9 +3688,18 @@ void Lowering::InsertPInvokeCallProlog(GenTreeCall* call)
         GenTree* frameAddr =
             new (comp, GT_LCL_VAR_ADDR) GenTreeLclVar(GT_LCL_VAR_ADDR, TYP_BYREF, comp->lvaInlinedPInvokeFrameVar);
 
+#ifdef _TARGET_X86_
+        // On x86 targets, indirect calls need the size of the stack args in InlinedCallFrame.m_Datum.
```

Comment:
indirect calls -> PInvoke calls. This applies to both direct and indirect calls.

Reply:
Yes. That's a copy-paste bug in the comment text. Thanks for pointing it out